docs/source/author/topics.rst

```diff
@@ -207,48 +251,52 @@ Adding a Programming Exercise
 You will now be able to add programming exercises to lessons by referencing the
 keys you specified in the programming exercises configuration file.
 
-.. note::
+A programming exercise is split into four markdown files, all of which are in
```

Comment:
This is incorrect as it may not be four files. The previous text was more accurate, there are four sections of files.

- One (and only one) exercise description file.
- Expected result for a specific language.
- Hints for a specific language.
- Solution for a specific language.

You cover this in more details later, but it shouldn't explicitly state four files.